css-borders-4/Overview.bs

```diff
@@ -141,6 +174,19 @@ Line Colors: the 'border-color' properties</h3>
 	If only one value is given,
 	it applies to both the <a>start</a> and <a>end</a> sides.
 
+	<wpt pathprefix="css/css-backgrounds/">
```

Comment:
I think that should be /css/css-backgrounds otherwise it is relative

Reply:
I made it absolute, though there is a wild mix of relative and absolute paths throughout the specs. And note that you also [used relative paths](https://github.com/w3c/csswg-drafts/blame/8ee7a05f87123313c9241f67449aef9b72b707ce/css-color-4/Overview.bs#L4822) for some tests.